src/ds/ds6/ds6-device.cpp

```diff
@@ -386,9 +369,6 @@ namespace librealsense
             val |= d400_caps::CAP_ROLLING_SHUTTER;  // e.g. ASRC
         if (0x2 == gvd_buf[depth_sensor_type])
             val |= d400_caps::CAP_GLOBAL_SHUTTER;   // e.g. AWGC
-        // Option INTER_CAM_SYNC_MODE is not enabled in D405
```

Comment:
Don't we need it for D500?
I see we use it for all devices but D405

Reply:
correct